src/templates/components/PaidForHeader.svelte

```diff
@@ -4,228 +4,185 @@
 	import Button from './Button.svelte';
 	import ArrowDown from './icons/ArrowDown.svelte';
 	import '$templates/components/fonts/SansBold.css';
-	import GuardianLabs from './icons/GuardianLabs.svelte';
+	import GuardianLabsLogo from './icons/GuardianLabsCircleLogo.svelte';
 
 	export let SeriesUrl: string;
 	export let ComponentTitle: string;
 	export let edition: Single['branding']['edition'];
-	export let templateType: 'single' | 'multiple';
 
 	export let popup = false;
 </script>
 
-<header class="{templateType}-header">
-	<div class="{templateType}-paid-content-and-title">
-		<div class="paid {templateType}-paid-content">
-			<strong>Paid content</strong>
-			<Button on:click={() => (popup = !popup)}
-				>About <ArrowDown width={12} flip={popup} /></Button
-			>
-			{#if popup}
-				<div id="popup">
-					<p>
-						Paid content is paid for and controlled by an advertiser and
-						produced by the Guardian Labs team
-					</p>
-					<a
-						class="popup-link"
-						href={`${CLICK_MACRO}https://theguardian.com/content-funding`}
-						target="_top">Learn more about Guardian Labs content &rarr;</a
-					>
-				</div>
-			{/if}
+<header class="header">
+	<div class="logo">
+		<GuardianLabsLogo {edition} />
+	</div>
+	<div class="paid-info-and-title">
+		<div class="paid-info">
+			<strong class="paid-info--label">Paid content</strong>
+			<div class="popup-container">
+				<Button on:click={() => (popup = !popup)}
+					>About <ArrowDown width={14} flip={popup} /></Button
+				>
+				{#if popup}
+					<div id="popup" class="popup">
+						<p class="popup-text">
+							Paid content is paid for and controlled by an advertiser and
+							produced by the Guardian Labs team
+						</p>
+						<a
+							class="popup-link"
+							href={`${CLICK_MACRO}https://theguardian.com/content-funding`}
```

Comment:
interesting that we don't use the `clickMacro` function here and other instances